categories/dnsimple.yaml

```diff
@@ -1,6 +1,10 @@
 Getting started:
 - "Getting Started"
-- "Getting Started With Your Personal Plan"
+- "Getting Started With Your Solo Plan"
```

Comment:
While this was here before for the Personal Plan, I think we should remove it unless we have a really good reason to keep it.